source/ota.c

```diff
@@ -86,182 +95,386 @@ typedef struct OtaStateTableEntry
     OtaState_t nextState;      /**< New state to be triggered*/
 } OtaStateTableEntry_t;
 
-/* OTA control interface. */
-
+/**
+ * @brief OTA control interface.
+ */
 static OtaControlInterface_t otaControlInterface;
 
-/* OTA data interface. */
-
+/**
+ * @brief OTA data interface.
+ */
 static OtaDataInterface_t otaDataInterface;
 
 /* OTA agent private function prototypes. */
 
-/* Called when the OTA agent receives a file data block message. */
-
+/**
+ * @brief Ingest a data block.
+ *
+ * A block of file data was received by the application via some configured communication protocol.
+ * If it looks like it is in range, write it to persistent storage. If it's the last block we're
+ * expecting, close the file and perform the final signature check on it. If the close and signature
+ * check are OK, let the caller know so it can be used by the system. Firmware updates generally
+ * reboot the system and perform a self test phase. If the close or signature check fails, abort
+ * the file transfer and return the result and any available details to the caller.
+ *
+ * @param pFileContext Information of file to be streamed.
+ * @param pRawMsg Raw job document.
+ * @param messageSize Length of document.
+ * @param pCloseResult Result of closing file in PAL.
+ * @return IngestResult_t IngestResultAccepted_Continue if successful, other error for failure
```

Comment:
Missing a period (.)

Reply:
Addressed in https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/116/commits/3e5b1d96211d212a79fd5aa4a1bd33a51add1e85